device/iot-device-client/src/main/java/com/microsoft/azure/sdk/iot/device/fileupload/FileUploadTask.java

```diff
@@ -119,7 +119,7 @@ public FileUploadTask(HttpsTransportManager httpsTransportManager)
     @Override
     public void run()
     {
-        Thread.currentThread().setName(THREAD_NAME);
+        Thread.currentThread().setName(THREAD_FIXED_NAME);
```

Comment:
The file upload task will be removed from our upcoming v2 release, so you can just skip renaming this thread

Reply:
Yeah, there are two file upload tasks, this one is related to getting the SAS uri, not the upload action. That one is deprecated and I didn't touch it.